Documentation/Products/UmbracoForms/Developer/Extending/Adding-a-Fieldtype.md

```diff
@@ -2,25 +2,34 @@
 
 *This builds on the "[adding a type to the provider model](Adding-a-Type.md)" chapter*
 
-Add a new class to the visual studio solution and make it inherit from Umbraco.Forms.Core.FieldType and override the Editor property.
+To add a custom field type create a new class in your Visual Studio solution. Your class will need `using Umbraco.Forms.Core;` in the Using directives at the top and to inherit from `Umbraco.Forms.Core.FieldType`.
 
-In the empty constructor add the following information:
+As an example, to create a new field type called CustomFormDataType, your class should contain the following information in it's constructor:
 
-	public Textfield() {
-		//Provider
-		this.Id = new Guid("D6A2C406-CF89-11DE-B075-55B055D89593 ");
-		this.Name = "Textfield";
-		this.Description = "Renders a html input";
-		this.Icon = "icon-autofill";
-        this.DataType = FieldDataType.String;
-		this.SortOrder = 10;
-	}
+    public class CustomFormDataType : Umbraco.Forms.Core.FieldType
```

Comment:
I think all ref's about DataType need to be changed to FieldType since they are very different things